src/app/test/mocks/dad_jokes_list_mock.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:app/access/dad_jokes/data/dad_joke_content_data.dart';
+
+import 'dad_jokes_data_mock.dart';
+
+List<DadJokeContentData> getDadJokesList() {
+  return mockedDadJokeResponse.dadJokeData.dadJokeChildrenData
+      .map(
+        (joke) => (joke.dadJokeContentData),
+      )
+      .toList();
+}
+
+List<DadJokeContentData> getFavoriteDadJokesList() {
+  return [
+    const DadJokeContentData(
+      id: "1cccv3s",
+      title: 'What is the first thing that Adam said to Eve in the Garden?',
+      selfText: 'You had better move back, I am not sure how big this thing gets.',
```

Comment:
I think we could use jokes that are more sfw in the mock data. 😅

Reply:
I didn't even see that 😅 cleary it isn't the most appropriate